tests/backstop/Dockerfile

```diff
@@ -1,33 +1,38 @@
-FROM node:22.14.0-bookworm
+FROM node:24.2.0-bookworm
```

Comment:
While here I updated the base image to the next LTS version of Debian and confirmed this has no effect.